app/src/androidTest/java/com/sdp/movemeet/viewTest/chatTest/ChatActivityTest.java

```diff
@@ -90,18 +81,17 @@ public void onFailure(@NonNull Exception e) {
 
         onView(withId(R.id.message_input_text)).perform(replaceText("my message"), closeSoftKeyboard());
 
-        onView(withId(R.id.button_send_message)).perform(click());
+        onView(withId(R.id.button_send_message)).perform(click()); //.perform(forceDoubleClick());
```

Comment:
Why we perform a double click ?

Reply:
I tried to perform a double click to ensure clicking on the "send button", but finally just left it as it is (a simple "perform(click())") and left the "forceDoubleClick()" in a comment for a quick access in case we have to replace this part of code!